### PR TITLE
BLD: Reenable conda updating

### PR DESCRIPTION
**Pull request by bashtage**
Reenable conda updateing in travis

### CODE REVIEW

File: statsmodels/tsa/tests/test_arima.py [hunk 1]
```diff
@@ -2092,7 +2092,10 @@ def test_arima111_predict_exog_2127():
     # rescaling results in convergence failure
     #model = sm.tsa.ARIMA(np.array(ef)*100, (1,1,1), exog=ue)
     model = ARIMA(ef, (1,1,1), exog=ue)
-    res = model.fit(transparams=False, iprint=0, disp=0)
+    res = model.fit(transparams=False, iprint=0, disp=0,
+                    maxiter=500)
+    import warnings
+    warnings.warn(res.params, SyntaxWarning)
```
Comment (josef-pkt) [inline statsmodels/tsa/tests/test_arima.py]: What is this SyntaxWarning?

Comment (bashtage) [reply statsmodels/tsa/tests/test_arima.py]: That is just to output parameters, temporarily.  This PR is failing now :-/

**Comment by bashtage** [comment]
@josef-pkt  This might as well go back.  It is just a reversion of a temporary change.

**Comment by josef-pkt** [comment]
I can merge if the problems are solved

Did you rebase on current master?
The `test_arima111_predict_exog_2127` failed again on one of the python versions and I thought I had finally gotten rid of those failures.

**Comment by bashtage** [comment]
Yes, rebates. Failure seems random. Passed on my Travis
https://travis-ci.org/bashtage/statsmodels/jobs/292710094

**Comment by bashtage** [comment]
I don't understand why this one test fails semi-regularly.  I can't see how it is related to this simple PR since the test environment is full of older packages.

**Comment by bashtage** [comment]
I'm am going to close this -- not worth the trouble.  It seems to fail on travis/statsmodels but passes on travis/bashtage.  It also passes on a local VM using the same packages from Conda. Sooner or later this conda will become the default in Miniconda and so this failure might crop up again.

**Pull request closed without merging**